docs/admin/kubelet-tls-bootstrapping.md

```diff
@@ -87,6 +177,19 @@ The flag to enable this bootstrapping when starting the kubelet is:
 --experimental-bootstrap-kubeconfig="/path/to/bootstrap/kubeconfig"
 ```
 
+Additionally, in 1.7 the kubelet implements __alpha__ features for enabling rotation of both its client and/or serving certs.
+These can be enabled through the respective `RotateKubeletClientCertificate` and `RotateKubeletServerCertificate` feature
+flags on the kubelet, but may change in backward incompatible ways in future releases.
```

Comment:
Also `RotateKubeletServerCertificate` on the API server. The auto approver for the server certificate is feature gated.